get-random-throw.js

```diff
@@ -0,0 +1,22 @@
+
+export function getRandomThrow(rockPaperScissor) {
+    if (rockPaperScissor === 1) {
+        return 'rock';
+    } else if (rockPaperScissor === 2) {
+        return 'paper';
+    } else {
+        return 'scissors';
+    }
+}
+
```

Comment:
remove this whitespace (you'll only ever need one line of whitespace, never two).